src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -223,10 +230,16 @@
            :fetch-routes #(fetch-routes % current-limit)}]
          [quo/bottom-actions
```

Comment:
@briansztamfater, what do you think is the cleanest way to pass along the bottom actions? 
Perhaps we just structure it so that the wrapper page has the bottom-actions rendered there?

e.g as children
```
(defn- view-internal
  []
  (let [send-bridge-data (rf/sub [:wallet/wallet-send])
        send-type        (:type send-bridge-data)]
    [rn/view {:style style/bridge-send-wrapper}
     [input-amount/view
      {:type send-type}
      [quo/bottom-actions ...]]
      ]))
 ```     
 or a view outside the input-amount

```
(defn- view-internal
  []
  (let [send-bridge-data (rf/sub [:wallet/wallet-send])
        send-type        (:type send-bridge-data)]
    [rn/view {:style style/bridge-send-wrapper}
     [input-amount/view
      {:type send-type}]
      [quo/bottom-actions ...]
      ]))
 ```     
 thoughts ?

Reply:
Not sure what you mean @J-Son89, I think you posted the same example twice. But I think it depends on how customizable we want it this component to be, I think if its just the buttons texts we want to customize we can have some simple props in the wrapper components.

Reply:
Sounds good!